app.js

```diff
@@ -3,16 +3,73 @@ require('dotenv').config();
 const express = require('express');
 const hbs = require('hbs');
 
-// require spotify-web-api-node package here:
+const SpotifyWebApi = require("spotify-web-api-node");
 
 const app = express();
 
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
-// setting the spotify-api goes here:
-
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+  });
+  
+  // Retrieve an access token
+  spotifyApi
+    .clientCredentialsGrant()
+    .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+    .catch(error => console.log('Something went wrong when retrieving an access token', error));
 // Our routes go here:
 
+app.get('/', (req, res) => {
+  console.log('index loading');
+  
+  res.render('index');
+});
+
+
+app.get('/artist-search', (req, res) => {
+  
+  spotifyApi
+    .searchArtists(req.query.artistName)
+    .then(apiResponse => {
+      console.log('finding artists');
```

Comment:
This is not really important here with the labs, but if you're creating projects - remember to remove all the console.logs so that the code is cleaner and more readable for others.